README.md

```diff
@@ -3,59 +3,72 @@
 
 ## go-sdk
 
-A Go SDK to make it easy to talk to the [Transloadit](https://transloadit.com) REST API.
+A GO Integration for [Transloadit](https://transloadit.com)'s file uploading and encoding service
```

Comment:
Please spell it just Go and not GO or Golang. That also how all official documentations refer to it.